buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
+    (with-ros-node ("inputer")
+        (let ((i 0) (pub (advertise "destination" "std_msgs/String")))
+            (loop-at-most-every .1
+                (publish-msg pub :data (read))))))
```

Comment:
does put take in more than just one arg? Do we need the named arg `:data`? Can't we just give `pub.publish-msg()` the `read` as an arg?